### PR TITLE
Don't move cards for external contributors

### LINKED ISSUE #26: Don't move cards for external constributors
Do not run action
* `MoveCardToReview`
* `MoveCardAfterReview`
when the PR is approved/requested by an external contributor.

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
Fixes #26

### CODE REVIEW

File: MoveCardAfterReview/ReadMe.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ jobs:
     # PRs from forks don't have required token authorization
     if: |
         github.event.pull_request.head.repo.full_name == github.repository
+        && github.event.review.author_association != 'NONE'
```
Comment (ilia-kebets-sonarsource) [inline MoveCardAfterReview/ReadMe.md:40]: Will this be true if the contributor is part of an organization other than SonarSource? Should the check be something like:

```
github.event.review.author_association != 'SonarSource'
```

Or whatever the id for SonarSource is.

**Review by ilia-kebets-sonarsource — APPROVED** [review]
LGTM, just check my single comment.

File: MoveCardAfterReview/ReadMe.md (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply MoveCardAfterReview/ReadMe.md:40]: The values here can be COLLABORATOR, CONTRIBUTOR, FIRST_TIMER, FIRST_TIME_CONTRIBUTOR, MANNEQUIN, MEMBER, NONE, OWNER
I played safe and excluded the one that has nothing to do with the repo, as there's a long list of weird values that we want to move on with.

Comment (ilia-kebets-sonarsource) [reply MoveCardAfterReview/ReadMe.md:40]: Thanks for explaining!

**Pull request merged**